package.json

```
@@ -15,7 +15,7 @@
     "lint": "lerna run lint",
     "cs-check": "lerna run cs-check",
     "cs-format": "lerna run cs-format",
-    "build": "lerna run --stream build",
+    "build": "lerna run --concurrency 2 --stream build",
```

I have a better idea for this change...
```suggestion
    "build": "lerna run --stream build",
    "build-serial": "lerna run --concurrency 1 --stream build",
```

---

Ooops, I didn't mean to commit this. I added it because Netlify was blowing up on my personal account

---

I agree about updating the docs :)

---

I'm hoping my next PR - switching to Turborepo - will solve the issue of only running the playground, but we'll see. 

Side note, have we considered switching from class components to functional components? React fast-refresh works better with them

---

There are some complex components that require some refactoring that I can imagine happening when we actually work on supporting React 18 ;)